projects/managed-hubs/support.md

```diff
@@ -173,24 +174,62 @@ This process is carried out in an ongoing basis by the {term}`Support Stewards`.
 (support:non-incident-response)=
 ### Non-incident response process
 
-1. **Respond within 24 working hours**. Acknowledge receipt of the support request and let the {term}`Community Representative` know about any investigation we have done thus far.
-2. **Spend 30 minutes trying to resolve**. If you believe you can resolve the issue within 30 minutes, try resolving it yourself.
-   1. If you resolve the issue, then jump to the "Communicate resolution" step.
-   2. If you don't believe you can resolve the issue in 30 minutes, jump to the next step.
-3. **Open an engineering issue**. If this is a {term}`Change Request` or {term}`Guidance Request` and you cannot resolve the issue within 30 minutes, then open a support issue for the team to discuss.
-   [{bdg-primary}`Open a general issue`](https://github.com/2i2c-org/infrastructure/issues/new?assignees=&labels=&template=01_new-issue.yml) that describes the request and any next steps we should take. Add the {guilabel}`support` label to it.
-4. **Communicate status**. Once we have an issue created to track the next steps, send a message to the Community Representative letting them know about the situation and what the next steps will be. Here's a template to help guide you:
+The goal of the non-incident response process is to bring standardization to our support response. This simple workflow tries to battle the bias towards a reactive response whereas it is also bringing some common patterns so all of our non-incident support responses are cohesive and shared among our support stewards.
+The current iteration of the workflow states each step and who should be responsible/accountable for the specific step, plus some other clarifications.
+
+When a new ticket lands in Freshdesk under the support group and it is not an incident, you should follow the following steps:
+
+1. `Who: Support steward`
+
+   **Respond within 24 working hours**. Acknowledge receipt of the support request and let the {term}`Community Representative` know about any investigation we have done thus far.
```

Comment:
```suggestion
   **Respond within 24 working hours**. Acknowledge receipt of the support request and let the {term}`Community Representative` know a time-boxed investigation will start soon. Please request any additional information you may need to be able to reproduce the issue in step 2.
```

@GeorgianaElena, WDYT?

Reply:
I like it very much! Thanks @damianavila